DependencyInjection/Compiler/AddProcessorsPass.php

```diff
@@ -25,16 +25,20 @@
  */
 class AddProcessorsPass implements CompilerPassInterface
 {
+    use PriorityTaggedServiceTrait;
+
     public function process(ContainerBuilder $container)
     {
         if (!$container->hasDefinition('monolog.logger')) {
             return;
         }
 
-        foreach ($container->findTaggedServiceIds('monolog.processor') as $id => $tags) {
+        foreach (array_reverse($this->findAndSortTaggedServices('monolog.processor', $container)) as $reference) {
```

Comment:
perhaps comment why `array_reverse` is used ?
otherwise its smart :)

Reply:
Good idea, done